libraries/ms-common/test/test_can.c

```diff
@@ -275,3 +278,49 @@ void test_can_default(void) {
   TEST_ASSERT_EQUAL(msg.msg_id, rx_msg.msg_id);
   TEST_ASSERT_EQUAL(msg.data, rx_msg.data);
 }
+
+#ifdef X86
+void test_can_x86_tx(void) {
+  volatile CanMessage rx_msg = { 0 };
+  CanMessage msg = {
+    .msg_id = 0xF,              //
```

Comment:
Any CAN message with ID below 14 (0xE) is critical, and 0xF=15 is a bit too close for comfort for me - if we ever increase the number of critical messages, this test will break. I assume you didn't use 0xFF because no ID can be greater than 64, but I'd prefer you use 63=0x3F or similar to make the test a bit less brittle. I'd also recommend you factor out this CAN message ID as a #define like `NON_CRITICAL_MSG_ID`.